gems/sorbet/test/snapshot/total/sorbet-runtime/expected/sorbet/rbi/sorbet-typed/lib/ruby/all/resolv.rbi

```diff
@@ -523,7 +523,7 @@ class Resolv
     end
 
     class Alt
-      Regex = Regex = T.let(T.unsafe(nil), Regexp)
```

Comment:
oof glad this is gone... do we know what caused this?

Reply:
This had been here since the very beginning—it shows up in pt's initial `sorbet-typed` commit—and predates me by a long time, so I have no idea. (I also got rid of it from `sorbet-typed` a month and a half ago, so we really haven't bumped this SHA in quite a while!)